Floofbot/Modules/Help.cs

```diff
@@ -57,73 +64,57 @@ public async Task HelpCommand()
                 });
                 fields.Clear();
             }
-            var pager = new PaginatedMessage
-            {
-                Pages = pages,
-                Color = Color.DarkGreen,
-                Content = Context.User.Mention,
-                FooterOverride = null,
-                ThumbnailUrl = Context.Client.CurrentUser.GetAvatarUrl(),
-                Options = PaginatedAppearanceOptions.Default,
-                TimeStamp = DateTimeOffset.UtcNow
-            };
-            await PagedReplyAsync(pager, new ReactionList
-            {
-                Forward = true,
-                Backward = true,
-                Jump = true,
-                Trash = true
-            });
+
+            string message = Context.User.Mention;
+            await PostHelpPages(message, pages);
         }
 
         [Summary("Show help for a specific module")]
         [Command("help")]
-        public async Task HelpCommand([Summary("module")]string requestedModule)
+        public async Task HelpCommand([Summary("module")] string requestedModule)
         {
-            List<string> moduleNames = new List<string>();
-            List<CommandInfo> commands = _commands.Commands.ToList();
-            List<EmbedFieldBuilder> fields = new List<EmbedFieldBuilder>();
-            List<PaginatedMessage.Page> pages = new List<PaginatedMessage.Page>();
-
-            // put available module names into a list instead of their object
-            foreach (ModuleInfo module in _commands.Modules.ToList())
-                moduleNames.Add(module.Name.ToLower());
-
-            if (!moduleNames.Contains(requestedModule.ToLower())) // no such cmd
+            List<string> moduleNames = _commandService.Modules.Select(x => x.Name.ToLower()).ToList();
+            if (!moduleNames.Contains(requestedModule.ToLower()))
             {
-                await Context.Channel.SendMessageAsync("Unable to find that command");
+                await Context.Channel.SendMessageAsync($"Unable to find the '{requestedModule}' module");
```

Comment:
I would put the word command instead of module as it will be users using this and they wont be aware of the concept of 'modules'

Reply:
Fixed. I also added an explanation for modules in the command summary because otherwise the user might start trying things like `.help ping` when it should be `.help utilities`.